dockerfiles/mssql2025-container-disk:c10s/install_mssql.sh

```
@@ -0,0 +1,32 @@
+#Install Mssql 2025  https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-red-hat?view=sql-server-ver17&tabs=rhel8%2C2025rhel10
+#Skip install ‘sudo yum install -y mssql-server-selinux’
+(choose: 7- Enterprise Core (PAID) - CPU Core utilization up to Operating System Maximum) - Yes/Yes/s3curePasswordString
```

Shellcheck (0.11.0)

[error] Couldn't parse this explicit subshell. Fix to allow more checks.

(SC1073)